chat-plugins/info.js

```diff
@@ -328,7 +328,7 @@ var commands = exports.commands = {
 	dt: 'details',
 	details: function (target) {
 		if (!target) return this.parse('/help details');
-		CommandParser.commands.data.apply(this, arguments);
+		this.run('data');
```

Comment:
Something more aesthetically pleasing would be ``this.run(commands.data)``.
However, that would cause issues in the offchance ``/data`` is monkeypatched via ``eval``.